src/types.ts

```diff
@@ -116,7 +117,12 @@ export type BaseAuth0APIClient = {
     getSecrets: (arg0: { id: string }) => Promise<Promise<Asset[]>>;
     addSecrets: (arg0: {}, arg1: Asset) => Promise<void>;
   };
-  logStreams: APIClientBaseFunctions;
+  logStreams: {
+    getAll: (arg0: SharedPaginationParams) => Promise<LogStream[]>;
+    create: (arg0: { id: string }) => Promise<LogStream>;
+    update: (arg0: {}, arg1: Partial<LogStream>) => Promise<LogStream>;
+    delete: (arg0: Asset) => Promise<void>;
+  };
```

Comment:
This applies stricter type the Auth0 Management API when working with log streams. This is important because I do a lot of checking against the `status` field so want to make sure it's correct.